MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -117,26 +129,27 @@ public void Logout (WebServiceLogin login)
 		}
 
 		[WebMethod]
-		public void CreateLanefile (WebServiceLogin login, int lane_id, string filename)
+		public void CreateLanefile (WebServiceLogin login, DBLane lane, string filename)
```

Comment:
Don't change the signature of existing API (methods that have [WebMethod]), instead add a new method with the signature you want.